server/server.go

```diff
@@ -165,7 +181,7 @@ func (s *Server) Start() (err error) {
 	return errors.Wrap(s.startAPIServer(), "failed to start API server")
 }
 
-// Stop will attempt to gracefully shut the Server down by signaling the stop
+// Stop will attempt to gracefully shut the Server down by signalling the stop
```

Comment:
Actually in US English, "signaling" is correct.

Reply:
Should be fixed in 5bef8a1.